### DISCUSSION TITLE
Introduction & questions before contribution.

### DISCUSSION THREAD

**Discussion by hexaltation** [AcademySoftwareFoundation/OpenTimelineIO#1463] [General]
Hello dear OTIO contributers,
With my collegue Aurelien Brelle, we were at the last otio meeting as auditors.
At woody-technologies we are interested to integrate OTIO as a timeline format converter in our products.
As a first step we are working with vizRT teams to interoperate our tools via OTIO conversion.
We are motivated to make some contributions effort.         
Especially in AAF adapter, audio/video effects support.
Implementation of more effects in core.
And where it can be useful.

We have some questions about ability to target/describe a particular stream in a clip edited in an audio track. Did we miss something or is it impossible in current version of code ?
It's seems that keyframe/controlPoints notion is by now absent from effects description. Is there some documents that blueprint it for future implementations ?
Thanks for your answers,
See you at the next meeting.
Grégoire Cutzach - (Woody Technologies)

**Comment by meshula** [comment]
Hi! Those are great questions, as a first step, I'd recommend browsing through existing discussions, as the use of curves in effects, and treatment of channels in audio are both hot topics. See, for example, 

https://github.com/AcademySoftwareFoundation/OpenTimelineIO/discussions/1009
https://github.com/AcademySoftwareFoundation/OpenTimelineIO/discussions/921

and others.

It would be great to move forward on both topics!
